cpu/msp430/cc2420-arch-sfd.c

```diff
@@ -45,6 +45,7 @@ ISR(TIMERB1, cc2420_timerb1_interrupt)
   ENERGEST_ON(ENERGEST_TYPE_IRQ);
   /* always read TBIV to clear IFG */
   tbiv = TBIV;
+  (void)tbiv;
```

Comment:
I suggest to either look for an alternative like defining `tbiv`as `volatile` or at least add a comment why this line was added.